webapp/admin_handler.go

```diff
@@ -15,38 +15,86 @@ import (
 	"github.com/web-platform-tests/wpt.fyi/shared"
 )
 
+func checkAdmin(acl shared.GitHubAccessControl, log shared.Logger, w http.ResponseWriter) bool {
+	if acl == nil {
+		http.Error(w, "Log in from the homepage first", http.StatusUnauthorized)
+		return false
+	}
+	admin, err := acl.IsValidAdmin()
+	if err != nil {
+		log.Errorf("Error checking admin: %s", err.Error())
+		http.Error(w, "Error checking admin", http.StatusInternalServerError)
+		return false
+	}
+	if !admin {
```

Comment:
This will never hit -  there is no case that `err == nil` but `admin == false`

Reply:
You're right; I'm fixing `IsValidAdmin`.